docs/application/native/guides/app-management/tidl.md

```diff
@@ -19,9 +19,13 @@ Usage:
   -o, --output=OUTPUT         The generated interface file.
   -p, --proxy                 Generate proxy code.
   -s, --stub                  Generate stub code.
+  -r, --rpclib                Generate C# library for rpc-port.
+  -t, --thread                Generate thread code with stub code.
+  -b, --beta                  Use beta version for private file sharing.
   -v, --version               Show version information.
 
 ```
+-t and -b options are supported since Tizen 6.0.
```

Comment:
For this sentence, I hope SRI-B reviewer to check and give the opinion to enhance it.
My suggestion is to apply the code tag to option words.

@annie-samsung Would you please assign a reviewer to check this PR?

```suggestion
Generating thread option with `-t` or `--thread`, and using beta version options with `-b` or `--beta` are supported since Tizen 6.0.
```

Reply:
Done

Reply:
@mijin85cho, the text looks good to me, however do you think we need to include it as a note tag?

Reply:
@annie-samsung I think note tag is acceptable at here. So do you want to change the form like below?


\> [!NOTE]
\> Generating thread option with \`-t\` or \`--thread\`, and using beta version options with \`-b\` or \`--beta\` are supported since Tizen 6.0.

Reply:
Yes, to get attention it is better to use Note tag.
@hjhun, recommend to use Note tag, as suggested by @mijin85cho

> [!NOTE]
> Generating thread option with `-t` or `--thread`, and using beta version options with `-b` or `--beta` are supported since Tizen 6.0.

Reply:
Done